packages/atomic/src/components/search/atomic-notification/atomic-notification.tsx

```diff
@@ -0,0 +1,85 @@
+import {Component, h, State, Fragment, Prop} from '@stencil/core';
+import {
+  NotifyTrigger,
+  NotifyTriggerState,
+  buildNotifyTrigger,
+} from '@coveo/headless';
+import {
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {Hidden} from '../../common/hidden';
+import {Bindings} from '../atomic-search-interface/atomic-search-interface';
+import InfoIcon from '../../../images/info.svg';
+import {AriaLiveRegion} from '../../../utils/accessibility-utils';
+import {Heading} from '../../common/heading';
+
+/**
+ * The `atomic-notification` component is responsible for displaying notifications generated by the Coveo Search API (see [Trigger](https://docs.coveo.com/en/1458)).
+ *
+ * @part notification - The wrapper around the notification.
+ * @part icon - The icon of the notification.
+ * @part text - The text of the notification.
+ */
+@Component({
+  tag: 'atomic-notification',
+  styleUrl: 'atomic-notification.pcss',
+  shadow: true,
+})
+export class AtomicNotifyTrigger implements InitializableComponent {
+  @InitializeBindings() public bindings!: Bindings;
+  protected notifyTrigger!: NotifyTrigger;
+
+  @BindStateToController('notifyTrigger')
+  @State()
+  private notifyTriggerState!: NotifyTriggerState;
+  @State() public error!: Error;
+
+  @AriaLiveRegion('notification') ariaMessage!: string;
+
+  /**
+   * The [heading level](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) to use above the notification, from 1 to 6.
+   */
+  @Prop({reflect: true}) public headingLevel = 0;
+
+  public initialize() {
+    this.notifyTrigger = buildNotifyTrigger(this.bindings.engine);
+  }
+
+  public renderNotification(text: string) {
+    return (
+      <div
+        part="notification"
+        class="shadow-lg bg-background border border-neutral-dark rounded-md p-6 flex items-center"
+      >
+        <atomic-icon
+          icon={InfoIcon}
```

Comment:
Suggestion: Make this icon a prop on the component, with the default value being the `InfoIcon`. It would be nice if there was also a way to say that you want "no icon" at all.